etc/adminhtml/system.xml

```diff
@@ -22,6 +22,11 @@
                        showInStore="1">
                     <label>Host url image service</label>
                 </field>
+                <field id="trim" translate="label" type="select" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Since you add a new field, you should also set a default in your config.xml file.